dune_client/api/extensions.py

```diff
@@ -123,6 +125,29 @@ def upload_csv(self, table_name: str, data: str, description: str = "") -> bool:
         except KeyError as err:
             raise DuneError(response_json, "UploadCsvResponse", err) from err
 
+    ##################
+    # Premium Features
+    ##################
```

Comment:
add "these features use APIs that are only available on paid subscription plans"